e2e-playwright/.env.dist

```diff
@@ -0,0 +1,3 @@
+PLAYWRIGHT_HOST=http://localhost:1337
+PLAYWRIGHT_USER_LOGIN=login
+PLAYWRIGHT_USER_PASSWORD=password
```

Comment:
please add new line at the end of file so we don't have that annoying red icon 😅